prueba_jaison.py

```diff
@@ -1,3 +1,8 @@
+name = raw_input("Enter a name")
+age = int(raw_input("Enter a age"))
+
+print("Your age is: ") + (name)
```

Comment:
Montro... donde tu aprendite a programar?

Mira:
print("Your age is: {}".format(name))